### ISSUE TITLE
Slider plugin in mobile view.

### ISSUE THREAD

**Issue by AwsomeCode** [grandnode/grandnode#911]
**Is your feature request related to a problem? Please describe.**
I am trying to use Slider plugin. Which is working fine in desktop or table mode. But in mobile view its getting hidden. Which also fine for some uses. But I still want to show the slider in Mobile View.

I have tried to dig deeper in to the source code but I haven't figured it out how its getting hidden in mobile view only.
If someone can show me the path I will be happy to send a pull request.

**Comment by PatrykPorabik** [comment]
In fact, it's simple CSS rule :) https://github.com/grandnode/grandnode/blob/develop/Plugins/Grand.Plugin.Widgets.Slider/Content/style.css#L194 Please insert your styling in this place. It will bring back the slider plugin. However, we would like to hide it by default, so pull request is not required.

**Comment by AwsomeCode** [comment]
@PatrykPorabik One of the first things I tried editing is style.css but no luck https://github.com/grandnode/grandnode/blob/develop/Plugins/Grand.Plugin.Widgets.Slider/Content/style.css#L194 .
Another thing I have tried is to editing slider.js but no luck.
Another thing I have done is clean the build by manually deleting the Bin&Obj folder.  but no luck.
Another thing I have done is I have created a sample plugin and attached it to the same zone of the slider plugin. Which is ok shown in all the device width.
